medmodels/treatment_effect/treatment_effect.py

```diff
@@ -110,10 +114,12 @@ def _set_configuration(
             outcome (Group): The group of outcomes to analyze.
             patients_group (Group, optional): The group of patients to analyze.
                 Defaults to "patients".
-            time_attribute (MedRecordAttribute, optional): The time attribute to use for
-                time-based analysis. Defaults to "time".
+            time_attribute (Optional[MedRecordAttribute], optional):  The time
+                attribute. If None, the treatment effect analysis is performed in an
+                static way (without considering time). Defaults to None.
             washout_period_days (Dict[str, int], optional): The washout period in days
-                for each treatment group. Defaults to dict().
+                for each treatment group. In the case of no time attribute, it is not
+                applied. Defaults to dict().
```

Comment:
Maybe we need a warning in the builder if a washout or grace period is set but no time attriubute? Otherwise the users may expect their settings to be applied even if they are not.

Reply:
Good point!

Reply:
Maybe it's possible to give this warning in the `_set_configuration() ` Function?